management/nodes/startupScripts/win2016-201700802-internal.ps1

```diff
@@ -0,0 +1,64 @@
+Set-ExecutionPolicy Unrestricted
+
+$machineSetupLog = 'C:\setup.log'
+$jenkinsServerUrl = $args[0]
+echo ('Jenkins Server URL: {0}' -f $jenkinsServerUrl) | Out-File -Append $machineSetupLog
```

Comment:
This idiom (echo "stuff"  | Out-File log.txt) is repeated over and over in this file.

It feels like the meat of this script should be in a function invoked like `Main-Function | Out-File log.txt` then all the Out-Files can be removed and just leave the Write-Output alone.